repair/repair.hh

```diff
@@ -357,7 +357,11 @@ struct repair_sync_boundary {
     public:
         tri_compare(const schema& s) : _pk_cmp(s), _position_cmp(s) { }
         int operator()(const repair_sync_boundary& a, const repair_sync_boundary& b) const {
-            int ret = _pk_cmp(a.pk, b.pk);
+            auto tmp = _pk_cmp(a.pk, b.pk);
+            auto ret = 0;
+            if (tmp != 0) {
+                ret = tmp < 0 ? -1 : 1;
+            }
```

Comment:
Instead of the four lines above, wouldn't just this work?
auto ret = tmp <=> 0

If it works, you don't even need to name tmp, just do

auto ret = _pk_cmp(a.pk, b.pk) <=> 0'

What am I missing?

Reply:
The return value of <=> cannot be assigned to an int.